src/main/java/org/fundacionjala/coding/ariel/Katas1.java

```diff
@@ -0,0 +1,52 @@
+package org.fundacionjala.coding.ariel;
+
+
+/**
+ * That class change.
+ */
+public class Katas1 {
+
+
+    private static final int COND = 5;
+    private String texto = "";
+    private String[] parts = texto.split(" ");
+    private String result = "";
+
+    /**
+     * That constructor init the class without args.
+     */
+    public Katas1() {
+        this.texto = "";
+    }
+
+    /**
+     * @param texto that constructor init the class with args
+     */
+    public Katas1(String texto) {
+        this.texto = texto;
+        this.parts = texto.split(" ");
+    }
+
+    /**
+     * @param texto That method rot a word that has five characters.
+     * @return return the word of five character but on reverse.
+     */
+    // change = new StringBuilder();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)